### PR TITLE
Clean up few little nits

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1178,7 +1178,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           on when it is invoked, and differ depending on which specific
           parameters are changed:</t>
          <t><list style="symbols">
-            <t>Any changes to the STUN/TURN servers to use affect
+            <t>Any changes to the STUN/TURN servers to use will affect
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: 'will' seems unnecessary here?

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -1250,8 +1250,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
-                <xref target="I-D.nandakumar-mmusic-proto-iana-registration"/>
+                the UDP/TLS/RTP/SAVPF <xref target="RFC5764"/>,
+		TCP/DTLS/RTP/SAVPF
+                <xref
+		    target="I-D.nandakumar-mmusic-proto-iana-registration"/>,
+		"UDP/DTLS/SCTP" <xref
+		    target="I-D.ietf-mmusic-sctp-sdp"/>, 
+            and "TCP/DTLS/SCTP" <xref
+		    target="I-D.ietf-mmusic-sctp-sdp"/>
                 RTP profiles.
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: 5764 is specific to DTLS-SRTP, right?

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: hmm - seems like we were missing a verb as I was thinking affect is noun but will defer to someone who speaks english

File: draft-ietf-rtcweb-jsep.xml (hunk 2 above)
Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: yah it defines UDP/TLS/RTP/SAVPF but not the other 3 profiles

**Pull request merged**